### PR TITLE
Removed deps on PageMessages

### PR DESCRIPTION
**Pull request by vileyne**

### CODE REVIEW

File: code/lightna/magento-os-frontend/js/common/AddToCart.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import { UserInput } from 'lightna/lightna-engine/lib/UserInput';
+import { PageMessage } from './PageMessage';
```
Comment (vesan-83) [inline code/lightna/magento-os-frontend/js/common/AddToCart.js:2]: we need to check if "./PageMessages" and "lightna/magento-os-frontend/PageMessages" makes difference for extends

Comment (vileyne) [reply code/lightna/magento-os-frontend/js/common/AddToCart.js]: it does, by alias replaced either `./PageMessages` or `lightna/magento-os-frontend/PageMessages`; depends how it's imported; so better not to use relative path like above

Comment (vesan-83) [reply code/lightna/magento-os-frontend/js/common/AddToCart.js]: ok, lets merge it and fire new ticket to change all to full path

**Review by vesan-83 — APPROVED** [review]

File: code/lightna/magento-os-frontend/js/common/AddToCart.js (hunk 1 above)
Comment (vileyne) [reply code/lightna/magento-os-frontend/js/common/AddToCart.js]: no need, we don't have such cases
(this is outdated thread)

**Pull request merged**